src/App.vue

```diff
@@ -18,17 +57,42 @@ import { Component, Vue } from 'vue-property-decorator'
 // Components
 import SbcHeader from 'sbc-common-components/src/components/SbcHeader.vue'
 import SbcFooter from 'sbc-common-components/src/components/SbcFooter.vue'
-import { EntityInfo } from '@/components/common'
+import SbcFeeSummary from 'sbc-common-components/src/components/SbcFeeSummary.vue'
+import { EntityInfo, Stepper, Actions } from '@/components/common'
+
+// Interfaces
+import { FilingDataIF } from '@/interfaces'
 
 @Component({
   components: {
     SbcHeader,
     SbcFooter,
-    EntityInfo
+    SbcFeeSummary,
+    EntityInfo,
+    Stepper,
+    Actions
   }
 })
-export default class App extends Vue {}
-</script>
+export default class App extends Vue {
+  private filingData: Array<FilingDataIF> = []
+  private totalFee: number = 0
+
+  private created (): void {
+    this.filingData.push({ filingTypeCode: 'OTINC', entityType: 'CP' })
```

Comment:
Should we start filingCodes and entityTypes ENUMs for these property values?

Reply:
Sure, I can add those.

Reply:
Done.